libp2p/src/main/kotlin/io/libp2p/mux/yamux/YamuxHandler.kt

```diff
@@ -10,19 +10,17 @@ import io.libp2p.etc.types.writeOnce
 import io.libp2p.etc.util.netty.ByteBufQueue
 import io.libp2p.etc.util.netty.mux.MuxChannel
 import io.libp2p.etc.util.netty.mux.MuxId
-import io.libp2p.mux.ClosedForWritingMuxerException
-import io.libp2p.mux.InvalidFrameMuxerException
-import io.libp2p.mux.MuxHandler
-import io.libp2p.mux.UnknownStreamIdMuxerException
-import io.libp2p.mux.WriteBufferOverflowMuxerException
+import io.libp2p.mux.*
 import io.netty.buffer.ByteBuf
 import io.netty.channel.ChannelHandlerContext
 import java.util.concurrent.CompletableFuture
 import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.atomic.AtomicBoolean
 import java.util.concurrent.atomic.AtomicInteger
 import kotlin.math.max
 import kotlin.properties.Delegates
 
+const val ACK_BACKLOG_ALLOWANCE = 256
```

Comment:
I would parametrize it for the `YamuxHandler` making this constant default value